source/hands_on/qtran.rst

```diff
@@ -0,0 +1,97 @@
+QTRAN
+^^^^^^^
+
+Overview
+---------
+QTRAN is proposed by Kyunghwan et al.(2019). QTRAN is a factorization method for MARL, which is free from such structural constraints and takes on a new approach to transforming the original joint action-value function into an easily factorizable one, with the same optimal actions.
+
+Compared to VDN(Sunehag et al. 2017), QMIX(Rashid et al. 2018), QTRAN guarantees more general factorization than VDN or QMIX, thus covering a much wider class of MARL tasks than does previous methods.
```

Comment:
add more details about `more general factorization`, for example you can describe some environments that qtran can be better than qmix